### DISCUSSION TITLE
How can I get all the data in the blog?

### DISCUSSION THREAD

**Discussion by namnguyenthanhwork** [facebook/docusaurus#9341] [Q&A] [answered]
I want all blog items to include some information below. I use some of this information to render recent posts on the homepage.

<img width="419" alt="image" src="https://github.com/facebook/docusaurus/assets/78526193/d4a796e8-ba81-4665-9774-125c203a4ce0">

I'm using `useBlogPost()` but I set `postsPerPage` so I can't get all.

Can everyone help me? Thank you.

**Comment by slorber** [comment, marked as answer]
I can't understand what you want to do exactly.

But maybe you'll find this example useful:
https://github.com/facebook/docusaurus/issues/4138#issuecomment-1703083513

**Reply by namnguyenthanhwork** [reply]
This is exactly what I want. Thanks for your help, Mr.@slorber.
